### ISSUE TITLE
Dashboard revision

### ISSUE THREAD

**Issue by wbieniek** [ansible/awx#6756]
##### ISSUE TYPE
Dashboard page enhancements 

##### SUMMARY
It would be nice to see a button on the Dashboard with currently running and pending  job numbers. How about alerting for longer the expecting running jobs?
Also last few jobs statistics  would be  nice as well.
Even better if there were list of items we could have we could customize our Dashboard to our needs.
I would reduce size of the graph as well.

**Comment by jlmitch5** [comment]
While it does not feature your requests, we have revamped the AWX dashboard in the new UI.

> It would be nice to see a button on the Dashboard with currently running and pending job numbers. How about alerting for longer the expecting running jobs?  Also last few jobs statistics would be nice as well.

While we aren't currently planning on including these features, job and template running/viewing/filtering functionality should be massively improved over the legacy dashboard, due to the fact the lists are fully-featured.

> Even better if there were list of items we could have we could customize our Dashboard to our needs.  I would reduce size of the graph as well.

We have discussed this a lot over the years and would love to have a configurable widget library for users to be able to customize their dashboard view, but the priority at the current moment with the new UI was getting as close to parity with the existing UI as possible.
